package.json

```diff
@@ -1,42 +1,42 @@
 {
   "name": "@snapshot-labs/overlord",
-  "version": "0.1.0-beta.0",
+  "version": "0.1.0",
+  "description": "JSON-RPC server for snapshot strategies token unit pricing in USD",
   "license": "MIT",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
   "scripts": {
     "build": "bun run tsc",
-    "lint": "bun run eslint ./src ./test ./scripts --ext .ts --fix",
-    "prepare": "bun run build",
-    "prepublishOnly": "bun run lint",
+    "typecheck": "bun run tsc --noEmit",
+    "lint": "bun run eslint ./src ./test --ext .ts",
+    "lint:fix": "bun run lint --fix",
     "test": "bun test",
-    "dev": "bun run build --watch",
-    "sync": "bun scripts/sync.ts"
+    "dev": "bun --watch src/index.ts",
+    "start": "node dist/index.js"
   },
   "eslintConfig": {
     "extends": "@snapshot-labs"
   },
   "prettier": "@snapshot-labs/prettier-config",
+  "engines": {
+    "node": ">=22.0.0",
+    "bun": ">=1.0.0"
+  },
   "dependencies": {
     "@ethersproject/contracts": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
-    "@ethersproject/wallet": "^5.7.0",
-    "@snapshot-labs/pineapple": "^0.2.0",
-    "bluebird": "^3.7.2",
-    "cross-fetch": "^3.1.5",
-    "mysql": "^2.18.1"
+    "cors": "^2.8.5",
+    "dotenv": "^17.2.1",
+    "express": "^5.1.0",
+    "zod": "^4.0.14"
   },
   "devDependencies": {
     "@snapshot-labs/eslint-config": "^0.1.0-beta.7",
     "@snapshot-labs/prettier-config": "^0.1.0-beta.7",
-    "@types/mysql": "^2.15.27",
-    "@types/node": "^18.11.9",
+    "@types/cors": "^2.8.17",
+    "@types/express": "^5.0.3",
+    "@types/node": ">=22.0.0",
     "eslint": "^8.28.0",
     "prettier": "^2.7.1",
-    "typescript": "^4.9.3"
-  },
-  "files": [
-    "dist/**/*",
-    "src/**/*"
-  ]
+    "supertest": "^7.1.4",
+    "typescript": "^5.8.0"
```

Comment:
Do our configs work with this version? I think those versions want TypeScript 4.x, maybe we should bump our configs to latest as we already have huge diff in this PR so all formatting changes won't be so noticable later.